### DISCUSSION TITLE
remote borg client

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbase/vorta#841] [Ideas]
for the use case of e.g. backing up a local server and still wanting a nice gui, vorta could invoke `ssh user@server borg ...` instead of just `borg ...`.

problem: all places using the file/directory selector dialog from the local OS / FS.

**Comment by ThomasWaldmann** [comment]
using vorta on the server would be an alternative to above. `ssh -X` or x2go.

**Comment by arthurlutz** [comment]
This could also mean generating borg commands that are setup remotely on the server so that a cron can handle the borg commands instead of being launched by vorta (or is this kind of feature a separate issue?)

**Comment by ThomasWaldmann** [comment]
@arthurlutz sound like a slightly different idea.

Also, generating commands for cron makes no difference whether you'ld edit these commands into your local or remote crontab. But I fear this is slightly out-of-scope of vorta anyway (and people who deal with crontabs and servers maybe don't need vorta to create the right commands anyway).
